CredentialProvider/MyVault/app/src/main/java/com/example/android/authentication/myvault/ui/GetPasskeyActivity.kt

```diff
@@ -112,7 +112,10 @@ class GetPasskeyActivity : FragmentActivity() {
 
         // Extract the requestJson and clientDataHash from this option.
         var clientDataHash: ByteArray? = null
-        if (request.callingAppInfo.origin != null) {
```

Comment:
`request.callingAppInfo.origin` is now "internal" and I don't know where to get that `privilegedAllowlist` from.

Reply:
https://developers.google.com/android/reference/kotlin/com/google/android/gms/identitycredentials/CallingAppInfo#origin()